src/index.ts

```diff
@@ -1,4 +1,13 @@
-console.log("[ITEM-STORE] Starting item store...");
+import { logger, logModel } from "./services/logger";
+
+logModel.service = "[ITEM-STORE]"
```

Comment:
this should now come from the config (SERVICE_NAME). so we can do logModel.service = `[${ SERVICE_NAME }]`